x-pack/plugins/security_solution/scripts/endpoint/common/stack_services.ts

```diff
@@ -117,7 +116,9 @@ export const createRuntimeServices = async ({
   let password = _password;
 
   if (asSuperuser) {
-    await waitForKibana(kibanaUrl);
+    await waitForKibana(
+      new KbnClient({ log, url: kibanaUrl, certificateAuthorities: [CA_CERTIFICATE] })
```

Comment:
Suggestion: use `createKbnClient()` instead to create the `KbnClient` instance instead and pass in the `noCertForSsl` param.

Reply:
Actually - def. use `createKbnClient()`. This approach here does **not** add credentials to the URL and assumes that URL already has the credentials in it.

Reply:
changed, Thank you @paul-tavares